tests/tests_python_parser.py

```diff
@@ -53,6 +53,24 @@ def test_parser_get_all_class_names(python_parser):
 		'AnotherClass'
 	])
 
+def test_parser_get_all_function_bodies(python_parser):
+	assert python_parser.get_all_function_bodies()["second_child"] == \
+		'def second_child[]:\n        """\n        This is second child\n        """\n        for i in range(10):\n            print(i)\n\n        return "Call me Liam"'
+	assert python_parser.get_all_function_bodies(get_index=True)["second_child"] == (
+		'def second_child[]:\n        """\n        This is second child\n        """\n        for i in range(10):\n            print(i)\n\n        return "Call me Liam"', 
+		(22, 8),
+		(28, 29)
+	)
+
+def test_parser_get_all_function_docstrings(python_parser):
+	assert python_parser.get_all_function_docstrings()["second_child"] == \
+		'"""\n        This is second child\n        """'
+	assert python_parser.get_all_function_docstrings(get_index=True)["second_child"] == (
+		'"""\n        This is second child\n        """', 
+		(22, 8), 
+		(28, 29)
```

Comment:
This sould give only where the doc string ends I think. Not where the function ends (like `28` in this case, which I believe the end line of the whole function) correct me if I am wrong please.

Reply:
Yes you are right, it's probably because the python parser didn't have a specific capture for the docstring but the capture for all the function